types/test.tsx

```diff
@@ -0,0 +1,38 @@
+// TypeScript Version: 3.2
+
+import * as React from 'react';
+import Layout from '@4c/layout';
+
+<Layout align="start" grow />;
+<Layout display="block" grow />;
+<Layout display="block" align="start" />; // $ExpectError
```

Comment:
this only errors against 3.2, which is why we specify 3.2 rather than 3.0 as elsewhere